src/components/symptom-shark/presentational/LogEntry/LogEntry.css

```diff
@@ -30,4 +30,9 @@
 .mdhui-ss-log-entry .mdhui-ss-notes-content {
     font-size: .88em;
     white-space: pre-wrap;
+}
+
+.mdhui-ss-log-entry .mdhui-ss-log-entry-action {
+    z-index: 1;
+    position: relative;
```

Comment:
I think you can drop the `position: relative` here.  It looks like it gets overridden with the same value by the unstyled button.